blockstore/splitstore/splitstore_compact.go

```diff
@@ -1296,7 +1338,7 @@ func (s *SplitStore) coldSetPath() string {
 	return filepath.Join(s.path, "coldset")
 }
 
-func (s *SplitStore) deadSetPath() string {
+func (s *SplitStore) discardSetPath() string {
 	return filepath.Join(s.path, "deadset")
```

Comment:
Just sanity checking - we're not un-renaming the function because deadSet is now not quite what it used to mean?

Reply:
Yeah it now has a use in compaction not just prune and this is better suited to that domain.